glean-core/android/src/main/java/mozilla/telemetry/glean/private/EventMetricType.kt

```diff
@@ -15,6 +15,7 @@ import mozilla.telemetry.glean.rust.toBoolean
 import mozilla.telemetry.glean.rust.toByte
 import org.json.JSONArray
 import org.json.JSONObject
+import mozilla.telemetry.glean.testing.ErrorType
```

Comment:
Our linter doesn't complain about unordered import statements here?
Consider me the linter: This line should be moved a bit higher to the other `mozilla.telemetry.glean` imports